.script/tests/KqlvalidationsTests/CustomTables/Netclean_Incidents_CL.json

```diff
@@ -0,0 +1,177 @@
+{
+    "Name": "Netclean_Incidents_CL",
+    "Properties": [
+        {
+            "Name": "TenantId",
+            "type": "string"
+        },
+        {
+            "Name": "SourceSystem",
+            "type": "string"
+        },
+        {
+            "Name": "MG",
+            "type": "string"
+        },
+        {
+            "Name": "ManagementGroupName",
+            "type": "string"
+        },
+        {
+            "Name": "TimeGenerated [UTC]",
+            "type": "string"
+        },
+        {
+            "Name": "Computer",
+            "type": "string"
+        },
+        {
+            "Name": "RawData",
+            "type": "string"
+        },
+        {
+            "Name": "Identifier_s",
+            "type": "string"
+        },
+        {
+            "Name": "foundTime_s",
+            "type": "string"
+        },
+        {
+            "Name": "creationTime_s",
+            "type": "string"
+        },
+        {
+            "Name": "lastAccessTime_s",
+            "type": "string"
+        },
+        {
+            "Name": "lastWriteTime_s",
+            "type": "string"
+        },
+        {
+            "Name": "agentInformatonIdentifier_s",
+            "type": "integer"
+        },
+        {
+            "Name": "Agentidentifier_s",
+            "type": "null"
+        },
+        {
+            "Name": "triggerSource_s",
+            "type": "string"
+        },
+        {
+            "Name": "domain_s",
+            "type": "string"
+        },
+        {
+            "Name": "domainName_s",
+            "type": "string"
+        },
+        {
+            "Name": "hasCollectedNearbyFiles_s",
+            "type": "boolean"
+        },
+        {
+            "Name": "externalIP_s",
```

Comment:
null datatype is not supported. It should be string as there is a _s at the end. Please make the required changes in other fields as well.